src/main/java/nl/tudelft/ti2206/group9/gui/scene/GameScene.java

```diff
@@ -100,6 +100,8 @@ public Parent createRoot() {
         keyBindings();
 
         CurrentItems.getSoundtrackPlayer().play();
+        CurrentItems.getSoundtrackPlayer().
+                setVolume(State.getSoundtrackVolume());
```

Comment:
It is indeed more 'logic', but not as we want it.
Explaining it is quite complicated but it has to do with initialising SoundtrackPlayers. The SoundtrackPlayer must first be initialised the first time, then it can be set. Not the other way around.
So if it was the other way around and you would start the application load an account with a different soundtrack volume level than the standard and you would play the game directly, the default sound effect volume will be started. This is not as we want it.
I hope that I made it clear :) If not, ask me in person ;)

<review,logic